README.md

```diff
@@ -143,6 +146,86 @@ Console.WriteLine("Authorize and Sign the Document" + signingLinks.Url);
 Console.WriteLine("Sign the Document" + signingLinks.AnonymousUrl);
 ```
 
+### <a name="create-freeform-invite"></a> Create a freeform invite to the document for signature
+
+*Freeform invite* - an invitation to sign a document which doesn’t contain any fillable fields.
+
+Simply upload a document and send it for signature right away. No need for adding fields and configuring roles. 
+Just add the signer's email address and customize the message in your email. 
+The document will be available for signing via a button in the email. 
+Clicking the button opens a document in SignNow editor. Signers can click anywhere on a document to add their signature.
+
+Remember: if your document contains even one fillable field, you have to create a role-based invite to get it signed.
+
+```csharp
+var invite = new FreeFormInvite("signer@signnow.com");
+
+// using `documentId` from the Upload document step
+// creating Invite request
+var inviteResponse = SignNow.Invites.CreateInviteAsync(documentId, invite).Result;
```

Comment:
change SignNow variable to signNowContext becasue it is confusing and could be considered as SignNow type with Invites static property.